apps/developer-hub/next.config.js

```diff
@@ -0,0 +1,77 @@
+import createBundleAnalyzer from "@next/bundle-analyzer";
+import { createMDX } from "fumadocs-mdx/next";
+
+const withAnalyzer = createBundleAnalyzer({
+  enabled: process.env.ANALYZE === "true",
+});
+
+const config = {
+  reactStrictMode: true,
+  turbopack: {
+    resolveExtensions: [".mdx", ".tsx", ".ts", ".jsx", ".js", ".mjs", ".json"],
+    rules: {
+      "*.svg": {
+        loaders: ["@svgr/webpack"],
+        as: "*.js",
+      },
+    },
+  },
```

Comment:
It doesn't look like you're actually using turbopack (I don't see `--turbo` in your `next dev` cli args) so I think you should either:

1. Remove this (it will become stale and incorrect if no one is using it)
2. Switch to using turbopack in dev

IMO 2 is preferable over 1